lkml/parser.py

```diff
@@ -330,9 +331,7 @@ def parse_csv(self) -> Optional[list]:
             self.logger.debug("%sTry to parse %s", self.depth * DELIMITER, grammar)
         values = []
 
-        if self.check(tokens.LiteralToken):
-            values.append(self.consume_token_value())
-        elif self.check(tokens.QuotedLiteralToken):
+        if self.check(tokens.LiteralToken) or self.check(tokens.QuotedLiteralToken):
```

Comment:
`check` accepts a variable number of tokens as positional args, so this can be condensed further to:

```self.check(tokens.LiteralToken, tokens.QuotedLiteralToken)```

Reply:
👍